tests/conftest.py

```diff
@@ -105,10 +106,13 @@ def _wait_for_file_to_be_available(integration_config: CommonConfig, workspace_n
     assert len(_get_file_names(integration_config, workspace_name)) > 0
 
 
-@pytest.fixture
-def workspace_name(integration_config: CommonConfig) -> str:
+@pytest.fixture(scope="session")
+def workspace_name(integration_config: CommonConfig) -> Generator[str, None, None]:
     """Create a workspace for the tests and delete it afterwards."""
-    workspace_name = "sdk_integration"
+    fake = Faker()
+    workspace_name = f"test_{'_'.join(fake.words(3))}"
```

Comment:
I did the same. I think this will lead to flakey tests.  What happens if an action is canceld or somehow does not clean up the state

Reply:
Okay thats fine. Lets try dynamic workspace names